addons/repair/models/repair.py

```diff
@@ -111,13 +111,20 @@ class Repair(models.Model):
     amount_tax = fields.Float('Taxes', compute='_amount_tax', store=True)
     amount_total = fields.Float('Total', compute='_amount_total', store=True)
     tracking = fields.Selection(string='Product Tracking', related="product_id.tracking", readonly=False)
+    invoice_state = fields.Selection(string='Invoice State', related='invoice_id.state')
+    invoice_count = fields.Integer(string='Invoices', compute='_compute_invoice_count')
 
     @api.depends('partner_id')
     def _compute_default_address_id(self):
         for order in self:
             if order.partner_id:
                 order.default_address_id = order.partner_id.address_get(['contact'])['contact']
 
+    @api.depends('invoice_id')
+    def _compute_invoice_count(self):
+        for order in self:
+            order.invoice_count = len(order.invoice_id)
```

Comment:
read_group please

Reply:
`invoice_id` is m2o, So it will be always 0 or 1 :thinking:

Reply:
yeah you're right i thought it was a o2m wrongly named.. @anr-odoo why do you need this comute?